src/app/features/task-repeat-cfg/store/task-repeat-cfg.reducer.spec.ts

```diff
@@ -6,7 +6,7 @@ const DUMMY_REPEATABLE_TASK: TaskRepeatCfg = {
   id: 'REPEATABLE_DEFAULT',
   title: 'REPEATABLE_DEFAULT',
   quickSetting: 'DAILY',
-  lastTaskCreation: 60 * 60 * 1000,
+  lastTaskCreation: 60 * 60 * 1000, // debug: Why this number? // todo: This number might cause issues with the tests later on…
```

Comment:
A date in the past is needed and `Date.now()` (the default value) won't work for the tests and because `0` doesn't yield valid dates. Do you have a suggestion maybe?

Reply:
Tbh. I don't know. I just left some of my first impression comments in, as they show possible issue sources. _(Like with the commented-out test)_

[[[ ~~I used "the day before today" in some of the tests. Though, that won't work very well for monthly and yearly.~~ Nvm, are set manually as well.
My monthly & yearly tests specify the `lastTaskCreation` themselves, however. So, there shouldn't be an issue with changing it _(or leaving it as is)_. ]]]